Tools/WinMLRunner/BindingUtilities.h

```diff
@@ -315,6 +343,112 @@ namespace BindingUtilities
         return ImageFeatureValue::CreateFromVideoFrame(videoFrame);
     }
 
+    ITensor CreateBindableTensorFromImage(
+        const ILearningModelFeatureDescriptor& description,
+        InputDataType inputDataType,
+        const CommandLineArgs args
```

Comment:
This is a sizeable object. const CommandLineArgs& would be better.

Reply:
Oops, missed this one.  Will definitely fix it.